src/nspawn/nspawn-mount.c

```diff
@@ -1336,3 +1340,57 @@ int setup_pivot_root(const char *directory, const char *pivot_root_new, const ch
 
         return r;
 }
+
+int pin_fully_visible_fs(void) {
+        int r;
+
+        (void) mkdir_p("/dev/.proc", 0755);
+        (void) mkdir_p("/dev/.sys", 0755);
```

Comment:
any particular reason to use /dev/ for this?

otherwise I think we should try to monopolize stuff that is primarily owned by the container manager and for what the container manager is at best a consumer in /run/host/ i guess.

Reply:
> any particular reason to use /dev/ for this?

I've traditionally used this but it really doesn't matter. Let's just use `/run/host`.